core/src/main/java/com/yahoo/ycsb/measurements/OneMeasurementHistogram.java

```diff
@@ -97,8 +128,11 @@ public synchronized void measure(int latency)
   @Override
   public void exportMeasurements(MeasurementsExporter exporter) throws IOException
   {
+    double mean = totallatency/((double)operations);
+    double variance = totalsquaredlatency/((double)operations) - (mean * mean);
     exporter.write(getName(), "Operations", operations);
-    exporter.write(getName(), "AverageLatency(us)", (((double)totallatency)/((double)operations)));
+    exporter.write(getName(), "AverageLatency(ms)", mean);
```

Comment:
please keep this in microseconds.

the variance should be in microseconds as well